common/src/IO/MapReader.cpp

```diff
@@ -219,13 +219,13 @@ namespace TrenchBroom {
                 return;
             }
 
-            const long rawId = std::atol(idStr.c_str());
-            if (rawId <= 0) {
+            const auto rawId = kdl::str_to_size(idStr);
```

Comment:
We should continue to reject id 0, removing this let the compatibility break bug slip in

Reply:
done! 43e3fe986b5a729abd3f93a24c89205d96fe5177